docs/zkapps/index.mdx

```diff
@@ -1,6 +1,15 @@
 ---
 title: Overview
```

Comment:
```suggestion
title: zkApp Overview
```

Reply:
Same with this sidebar change as well. It will go from `Overview` -> `zkApp Overview` in the sidebar.

Reply:
okay, same comment as above, I found out how, we add

`sidebar_label: Overview`

I wonder why we use `hide_title: true` and then rely on the `# Heading` for the title?

Reply:
That's a good question! I think the reason for this is to have the `Info` box as the very first thing that's shown on the page. If we use the normal title, the title is shown first with the `Info` box shown below it. By using `hide_title`, and then manually specifying the title, we can show the `Info` box right away.

`Info` box specifically is that box that states
`Please note that zkApp programmability is not yet available on Mina Mainnet, but zkApps can now be deployed to Berkeley Testnet.`